### PR TITLE
Feat/644 bug data sort by axis

### LINKED ISSUE #644: [Bug] data sort by axis

### PR DESCRIPTION
**Pull request by xuefei1313**
[[中文版模板 / Chinese template](https://github.com/VisActor/VChart/blob/main/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [x] New feature
- [ ] Bug fix
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Refactoring
- [ ] Update dependency
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Release
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Workflow
- [ ] Other (about what?)

### 🔗 Related issue link

close #644 

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

---

### 🚀 Summary

copilot:summary

### 🔍 Walkthrough

copilot:walkthrough

### CODE REVIEW

File: packages/vchart/src/series/cartesian/interface.ts [hunk 1]
```diff
@@ -32,6 +32,12 @@ export interface ICartesianSeriesSpec extends ISeriesSpec {
    * 用于3d散点图等
    */
   zField?: string | string[];
+
+  /** 是否将数据按照数轴排序
+   * @default false
+   * @since 1.3.0
+   */
+  sortDataByAxis?: boolean;
```
Comment (kkxxkk2019) [inline packages/vchart/src/series/cartesian/interface.ts:40]: 文档上也补充下吧

**Review by kkxxkk2019 — APPROVED** [review]

**Comment by kkxxkk2019** [comment]
我 merge 来的 dev/1.3.0 发现 bad case 蛮多的

File: packages/vchart/src/series/cartesian/interface.ts (hunk 1 above)
Comment (xuefei1313) [reply packages/vchart/src/series/cartesian/interface.ts:40]: ok

**Comment by xuefei1313** [comment]
rebase error , use pull 704 replace

**Pull request closed without merging**